docs/book/introduction.md

```diff
@@ -0,0 +1,135 @@
+---
+description: Welcome to ZenML!
+---
+
+# ⭐ Introduction
+
+**ZenML** is an extensible, open-source MLOps framework for creating portable, production-ready machine learning pipelines. By decoupling infrastructure from code, ZenML enables developers across your organization to collaborate more effectively as they develop to production.
+
+<div align="center" data-full-width="true">
+
+<figure><img src=".gitbook/assets/intro-zenml-overview.png" alt="ZenML overview"><figcaption><p>An overview of ZenML</p></figcaption></figure>
```

Comment:
Why are we using HTML here vs Markdown?

Reply:
I think whenever we tried to scale down the image inside of gitbook markdown was used. Problem with this is that when we move the files withint pycharm/vscode, this would not be updated in code and break the docs. I have already started finding instances of this and replacing with the appropriate markdown format.